centreon/tests/e2e/features/HostGroups/01-host-group-configuration/index.ts

```diff
@@ -0,0 +1,187 @@
+            jsonName: 'admin',
+            loginViaApi: false
+        });
+    });
+
+When('a host group is configured',
+    () => {
+        cy.addHostGroup({
+            name: hostGroup
+        });
+
+        cy.addHost({
+            hostGroup: hostGroup,
+            name: services.serviceOk.host,
+            template: 'generic-host'
+        })
+            .addService({
+                activeCheckEnabled: false,
+                host: services.serviceOk.host,
+                maxCheckAttempts: 1,
+                name: services.serviceOk.name,
+                template: services.serviceOk.template
+            })
+            .addService({
+                activeCheckEnabled: false,
+                host: services.serviceOk.host,
+                maxCheckAttempts: 1,
+                name: services.serviceWarning.name,
+                template: services.serviceWarning.template
+            })
+            .addService({
+                activeCheckEnabled: false,
+                host: services.serviceOk.host,
+                maxCheckAttempts: 1,
+                name: services.serviceCritical.name,
+                template: services.serviceCritical.template
+            }).applyPollerConfiguration();
+
+        checkHostsAreMonitored([
+            { name: services.serviceOk.host },
+        ]);
+        checkServicesAreMonitored([
+            { name: services.serviceCritical.name },
+            { name: services.serviceOk.name }
+        ]);
+        cy.submitResults(resultsToSubmit);
+    });
+
+When('the user changes some properties of the configured host group "name, alias, comments"',
+    () => {
+        cy.navigateTo({
+            page: "Host Groups",
+            rootItemNumber: 3,
+            subMenu: "Hosts",
+        });
+        cy.wait('@getTimeZone');
+        cy.getIframeBody().contains(hostGroup).click({
+            force: true,
+        });
+
+        cy.waitUntil(
+            () => {
+                return cy
+                    .getByLabel({ label: 'Up status hosts', tag: 'a' })
+                    .invoke('text')
+                    .then((text) => {
+                        if (text !== '2') {
+                            cy.exportConfig();
+                        }
+
+                        return text === '2';
+                    });
+            },
+            { interval: 20000, timeout: 100000 }
+        );
+
+        cy.getIframeBody().find('input[name="hg_name"]').clear().type(hostGroup + '_new');
+        cy.getIframeBody().find('input[name="hg_alias"]').clear().type(hostGroup + '_new');
+        cy.getIframeBody().find('textarea[name="hg_comment"]').clear().type('Host group updated for test!');
+        cy.getIframeBody().find('input.btc.bt_success[name^="submit"]').eq(0).click();
+        cy.wait('@getTimeZone');
+    }
+);
+
+Then('the properties are updated',
+    () => {
+        cy.exportConfig();
+        cy.getIframeBody().contains(hostGroup + '_new').should('exist');
```

Comment:
where do you check that properties are updated ?

Reply:
after update, the name of the host group will be hostGroup + '_new, so in the page should contains hostGroup + '_new